apps/communications/dialer/test/unit/keypad_test.js

```diff
@@ -408,6 +422,15 @@ suite('dialer/keypad', function() {
         subject.init(/* oncall */ false);
       });
 
+      test('Disable abbreviated dialing codes', function() {
+        this.sinon.spy(KeypadManager, '_getSpeedDialNumber');
+
+        setupAbbreviatedDialingCodesMocks();
+        typeAbbreviatedDialingCode('1#');
```

Comment:
nit: We could repeat this same test for the MMI code, but it wouldn't really force us down an extra code path.